bundles/k3d/demo/uds-bundle.yaml

```diff
@@ -0,0 +1,157 @@
+  - name: dev-namespaces
+    path: ../../../
+    ref: 0.1.0
+
+  - name: postgres-operator
+    repository: ghcr.io/defenseunicorns/packages/uds/postgres-operator
+    # x-release-please-start-version
+    ref: 1.10.1-uds.4-upstream
+    # x-release-please-end
+    overrides:
+      postgres-operator:
+        uds-postgres-config:
+          variables:
+            - name: POSTGRESQL
+              description: "Configure postgres using CRs via the uds-postgres-config chart"
+              path: postgresql
+
+
+  - name: dev-redis
+    repository: ghcr.io/defenseunicorns/packages/uds/dev-redis
+    ref: 0.0.1
+
+  - name: dev-secrets
+    path: ../../../
+    ref: 0.1.0
+    exports:
+      - name: ACCESS_KEY
+      - name: SECRET_KEY
+      - name: DB_PASSWORD
+
+  - name: gitlab
+    repository: ghcr.io/defenseunicorns/packages/uds/gitlab
+    # x-release-please-start-version
+    ref: 16.9.1-uds.1-upstream
+    # x-release-please-end
+    overrides:
+      gitlab:
+        gitlab:
+          variables:
+            - name: MIGRATIONS_RESOURCES
+              description: "Gitlab Migrations Resources"
+              path: "gitlab.migrations.resources"
+            - name: WEBSERVICE_REPLICAS
+              description: "Gitlab Webservice Min Replicas"
+              path: "gitlab.webservice.minReplicas"
+            - name: WEBSERVICE_RESOURCES
+              description: "Gitlab Webservice Resources"
+              path: "gitlab.webservice.resources"
+            - name: WORKHORSE_RESOURCES
+              description: "Gitlab Workhorse Resources"
+              path: "gitlab.workhorse.resources"
+            - name: SIDEKIQ_REPLICAS
+              description: "Gitlab Sidekiq Min Replicas"
+              path: "gitlab.sidekiq.minReplicas"
+            - name: SIDEKIQ_RESOURCES
+              description: "Gitlab Sidekiq Resources"
+              path: "gitlab.sidekiq.resources"
+            - name: REGISTRY_REPLICAS
+              description: "Gitlab Registry Min Replicas"
+              path: "registry.hpa.minReplicas"
+            - name: SHELL_REPLICAS
+              description: "Gitlab Shell Min Replicas"
+              path: "gitlab.gitlab-shell.minReplicas"
+            - name: POSTGRES_USERNAME
+              description: "Gitlab Postgres Username"
+              path: "global.psql.username"
+              default: "gitlab.gitlab"
+            - name: POSTGRES_SECRET_REF
+              description: "Gitlab Postgres Password Secret Ref"
+              path: "global.psql.password.secret"
+              default: "gitlab.gitlab.pg-cluster.credentials.postgresql.acid.zalan.do"
+
+  - name: gitlab-runner
+    repository: ghcr.io/defenseunicorns/packages/uds/gitlab-runner
+    ref: 16.8.0-uds.1-upstream
+
+  - name: sonarqube
+    repository: ghcr.io/defenseunicorns/packages/uds/sonarqube
+    # x-release-please-start-version
+    ref: 8.0.3-uds.4-upstream-ce
+    # x-release-please-end
```

Comment:
renovate instead of release-please

Reply:
Yeah copy/paste goodness when i started the bundle in the first place. I think the correct answer is delete though right? Your renovate config should pick these up automatically?